src/main/java/io/boomerang/service/refactor/TaskServiceImpl.java

```diff
@@ -536,11 +536,14 @@ private void finishWorkflow(ActivityEntity activity) {
     boolean workflowCompleted = dagUtility.validateWorkflow(activity);
     
     if (activity.getStatusOverride() != null) {
+      //Workflow activity changed to the override status
```

Comment:
What "override" status exactly means? Have never seen this in the UI.

Reply:
statusOverride is applicable for the `Set Workflow Result Status` task template